apiary.apib

```diff
@@ -4421,6 +4422,8 @@ Get a list of available payment terms.
             + (object)
                 + id: `c93ddb52-0af8-47d9-8551-441435be66a7` (string)
                 + Include PaymentTerm
+        + meta (object)
+            + `default`: `c93ddb52-0af8-47d9-8551-441435be66a7` (string)
```

Comment:
I've actually been having some doubts for the past couple of days about the naming, should we include what the default is about?

```
default_payment_term: `c93ddb52-0af8-47d9-8551-441435be66a7`
```
Or apply more structure
```
default: {
  id: c93ddb52-0af8-47d9-8551-441435be66a7,
  type: paymentTerm
}
```

What if we introduce more "default" a-like values?

Reply:
Or apply more structure through
```
default: {
  id: c93ddb52-0af8-47d9-8551-441435be66a7,
  type: paymentTerm
}
```

Reply:
Or
```
default_payment_term: {
  id: c93ddb52-0af8-47d9-8551-441435be66a7,
  type: paymentTerm
}
```

Reply:
I'm actually fine with the original. My reasoning: We're on a `.list` endpoint and will return a single list (maybe paginated) of items of a single type. So `default` is for me clearly the default item from that list.

Reply:
We decided on keeping as is.